scripts/release/prerelease.ts

```diff
@@ -97,6 +103,8 @@ export async function runPrerelease({
   }
 }
 
+const FORBIDDEN_TAGS = ['latest'];
```

Comment:
Should we add "next", "exp", and/or "canary"? They're not as important but, I mean, since the code is all set up, why not?